api/src/pcapi/utils/requests.py

```diff
@@ -23,6 +23,14 @@
 REQUEST_TIMEOUT_IN_SECOND = 10
 
 
+class ExternalAPIException(Exception):
+    is_retryable: bool
+
+    def __init__(self, is_retryable: bool, *args: object) -> None:
+        self.is_retryable = is_retryable
+        super().__init__(*args)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Pas de kwargs pour les Exceptions ?

Reply:
Pas trop d'idée là dessus, peut-être @dbaty tu as la réponse ?

Reply:
Apparemment, pas nécessaire : 

```python
>>> Exception(foo="bar")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Exception() takes no keyword arguments
```